requirements-sdp.txt

```diff
@@ -0,0 +1,14 @@
+asdf==2.3.3
```

Comment:
So does this entire file need to be reviewed and updated periodically when any versions change in the DMS environment?

Reply:
Yes, that would be the idea.  We will only move forward on versions in the pipeline when we decide to, rather than having conda introduce an update outside of our control.    This does mean that at the start of a build that this file should be reviewed and updated accordingly.

Reply:
We will be testing both the pinned dependencies in the `requirements` file and the rolling, released dependencies.  How often we want to do each may depend on which part of the dev cycle we're in.

It may be that we want to not use a `requirements` file now for the daily builds, but use one for the build on the weekends.  And then switch that up as we get a month or so out from when we start having to have our first rc1 ready for Build 7.4.  But we can discuss this further and see what gives us the most flexibility and stability.